src/utils/polygon.h

```diff
@@ -991,9 +992,40 @@ class Polygons
         }
     }
 
-    void fixSelfIntersections()
+    void resize(const coord_t& numerator, const coord_t divisor)
     {
-        ClipperLib::SimplifyPolygons(paths);
+        if (numerator == 1 && divisor == 1)
+        {
+            return;
+        }
+
+        Polygons& thiss = *this;
+        for (size_t p = 0; p < size(); p++)
+        {
+            for (Point& pt : thiss[p])
+            {
+                pt.X = (pt.X * numerator) / divisor;
+                pt.Y = (pt.Y * numerator) / divisor;
+            }
+        }
+    }
+
+    void translate(const Point vec)
```

Comment:
Duplicates the `PolygonRef::translate` function.

Reply:
I'll have to look into this, since I'm not the original author. I think @rburema wrote it, maybe he can comment on this.

Reply:
You must have duplicated this function from Rburema's original code then, since there is no `translate` function that you remove in a different location. The functionality is already present anyway in `PolygonRef` so it should be removed from here or from there, or one should call the other.

Reply:
I could be wrong but from what I understand is that they're two different classes, I believe that this one is a translation for the collection of PolygonRef(s).

I'm gonna change it to:
```
    void translate(const Point vec)
    {
        if (vec.X == 0 && vec.Y == 0)
        {
            return;
        }

        for (PolygonRef poly : *this)
        {
            poly.translate(vec);
        }
    }
```